examples/single-sign-on/okta/okta.tf

```diff
@@ -0,0 +1,155 @@
+# You can generate an Okta API token in the Okta Developer Console. Follow these instructions: https://bit.ly/get-okta-api-token
```

Comment:
Same note regarding usage of teams module and showing how groups can be tied to namespaces?

Reply:
Done! :)